### PR TITLE
Fix download badge in README.md file and a typo

### PR DESCRIPTION
**Pull request by atharva-satpute**
### Summary
Fixed the download badge and a typo in the README file

### CODE REVIEW

**Comment by atharva-satpute** [comment]
Duplicate pull request of  #10661

File: README.md [hunk 1]
```diff
@@ -5,7 +5,7 @@
 [![Coverage Status](https://coveralls.io/repos/github/Qiskit/qiskit-terra/badge.svg?branch=main)](https://coveralls.io/github/Qiskit/qiskit-terra?branch=main)
 ![PyPI - Python Version](https://img.shields.io/pypi/pyversions/qiskit)
 [![Minimum rustc 1.64.0](https://img.shields.io/badge/rustc-1.64.0+-blue.svg)](https://rust-lang.github.io/rfcs/2495-min-rust-version.html)
-[![Downloads](https://pepy.tech/badge/qiskit-terra)](https://pypi.org/project/qiskit-terra/)<!--- long-description-skip-end -->
+[![Downloads](https://static.pepy.tech/badge/qiskit)](https://pypi.org/project/qiskit-terra/)<!--- long-description-skip-end -->
```
Comment (1ucian0) [inline README.md:8]:
```suggestion
[![Downloads](https://static.pepy.tech/badge/qiskit-terra)](https://pepy.tech/project/qiskit-terra)<!--- long-description-skip-end -->
```

File: README.md [hunk 2]
```diff
@@ -5,7 +5,7 @@
 [![Coverage Status](https://coveralls.io/repos/github/Qiskit/qiskit-terra/badge.svg?branch=main)](https://coveralls.io/github/Qiskit/qiskit-terra?branch=main)
 ![PyPI - Python Version](https://img.shields.io/pypi/pyversions/qiskit)
 [![Minimum rustc 1.64.0](https://img.shields.io/badge/rustc-1.64.0+-blue.svg)](https://rust-lang.github.io/rfcs/2495-min-rust-version.html)
-[![Downloads](https://static.pepy.tech/badge/qiskit-terra)](https://pepy.tech/project/qiskit-terra)<!--- long-description-skip-end -->
+[![Downloads](https://img.shields.io/pypi/dm/qiskit-terra.svg)](https://pypi.org/project/qiskit-terra/) <!--- long-description-skip-end -->
```
Comment (1ucian0) [inline README.md]: Undo this change.
```suggestion
[![Downloads](https://static.pepy.tech/badge/qiskit-terra)](https://pepy.tech/project/qiskit-terra)<!--- long-description-skip-end -->
```

**Review by 1ucian0 — APPROVED** [review]
Thanks for fixing the typo!
